test/unit/rules/lint-img-alt-attributes-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'img-alt-attributes',
+
+  config: true,
+
+  good: [
+    '<img alt="hullo">',
+    '<img src="hello.jpg" alt="">'
```

Comment:
@Robdel12 I imagine a framework for required attributes for certain elements will probably be an eventual solution. E.g. like in axe-core:

https://github.com/dequelabs/axe-core/blob/4ef74435ac92a88c602690ac1e243c404f750d00/lib/commons/aria/index.js#L159